Test/Test/ConsoleReader.h

```diff
@@ -1,30 +1,13 @@
 #pragma once
-#include <iostream>
-#include <conio.h>
+
+const char esc = 27;
```

Comment:
We agreed to keep these constants in some class. Maybe it is implemented in further PRs. If not - then move them

Reply:
Yes, it is implemented in later PR